modules/programs/senpai.nix

```diff
@@ -61,13 +97,34 @@ in {
         <manvolnum>5</manvolnum></citerefentry>.
       '';
     };
+    extraConfig = mkOption {
+      type = types.attrs;
+      default = { };
+      description = ''
+        Options that should be appended to the senpai configuration file.
+      '';
+    };
   };
 
   config = mkIf cfg.enable {
+    assertions = with cfg.config; [
+      {
+        assertion = !isNull password-cmd -> isNull password;
+        message = "senpai: password-command overrides password!";
+      }
+      {
+        # TODO: remove me for the next release
+        assertion = isNull no-tls;
+        message = ''
+          Specifying 'senpai.no-tls' is deprecated,
+          set 'senpai.extraConfig = { tls = false; }' instead.
+        '';
+      }
+    ];
     home.packages = [ cfg.package ];
-    xdg.configFile."senpai/senpai.yaml".source =
-      cfgFmt.generate "senpai.yaml" cfg.config;
+    xdg.configFile."senpai/senpai.scfg".text =
+      lib.hm.generators.toSCFG (cfg.config // cfg.extraConfig);
   };
 
-  meta.maintainers = [ hm.maintainers.malvo ];
+  meta.maintainers = [ maintainers.jleightcap ];
```

Comment:
Does @malte-v not want to be a maintainer any more?

Reply:
that is correct.